sig-policy/policy-charter.md

```diff
@@ -0,0 +1,35 @@
+# SIG Policy charter
+
+## Scope
+
+SIG Policy covers governance capability, security remediation, and configuration management for the Open Cluster Management (OCM) project.
```

Comment:
I think we might want a clearer definition on what is policy, what the use cases it intends to resolve. Some examples would be nice.

Reply:
I added more details. Does this cover what you had in mind?